docs/source/index.rst

```diff
@@ -81,7 +81,6 @@ Features described in this documentation are classified by release status:
    torch.autograd <autograd>
    torch.library <library>
    cuda
-   torch._awaits <_awaits>
```

Comment:
We can probably leave this one in place as @svekars is working on PR to mark everything that starts with `_` as beta